Wrappers/Python/ccpi/web_viewer/dev-environment.yml

```diff
@@ -1,11 +1,11 @@
 # run: conda env create --file environment.yml
-name: cilviewer_webapp
+name: cilviewer_webapp_trame2
```

Comment:
I don't like this renaming it adds unnecessary length to the name that you'd have to type.